amo_storage.py

```diff
@@ -9,22 +9,28 @@
 db = SQLAlchemy()
 redis = Redis()
 ceph = CephAdapter()
-DEFAULT_CONFIG_PATH = "config.ini"
+
+DEFAULT_CONFIG_FILENAME = "config.ini"
+DEFAULT_KEY_FILENAME = "key.json"
+DEFAULT_CONFIG_DIR = "configurations"
```

Comment:
Let's make it shorter. Please make it "config".

Reply:
I pushed the commit with your comment reflected